mysql-test/include/percona_get_last_bmp_file_name_size.inc

```diff
@@ -0,0 +1,62 @@
+# Purpose:
+# To get name and size of the latest file written
+# by online log tracking. In additional it returns
+# number of bitmap files found in the provided directory.
+#
+# Parameters:
+# --let $bmp_data_dir = path/to/bmp/files/dir
```

Comment:
for InnoDB related stuff, the inc file should be in mysql-test/suite/innodb/include

Reply:
Done